index.js

```diff
@@ -247,4 +248,86 @@ function getCountStr(count) {
   }
 }
 
+// GraphQL query for LC daily question
+const dailyLCQuery = `
+query questionOfToday {
+        activeDailyCodingChallengeQuestion {
+            date
+            userStatus
+            link
+            question {
+                acRate
+                difficulty
+                freqBar
+                frontendQuestionId: questionFrontendId
+                isFavor
+                paidOnly: isPaidOnly
+                status
+                title
+                titleSlug
+                hasVideoSolution
+                hasSolution
+                topicTags {
+                    name
+                    id
+                    slug
+                }
+            }
+        }
+    }
+`;
+
+// POST request to get LC daily question
+const getLCQuestion = () => {
+  return axios
+  ({
+    url: 'https://leetcode.com/graphql',
+    method: 'post',
+    headers: {
+      'content-type': 'application/json',
+    },
+    data: {
+      query: dailyLCQuery,
+    },
+  });
+};
+
+let cronJob;
+
+// Command to start cron job
+// Definitely need to change this to an admin-only command
+bot.onText(/\/startDailyLCSchedule/, (msg) => {
+  const chatId = msg.chat.id;
+  const reply = `Starting daily LC schedule.`;
+  bot.sendMessage(chatId, reply);
+  console.log('start cron job');
+  // Just for testing every 5 seconds
+  // cronJob = cron.schedule('*/5 * * * * *', () => {
+  // Posts a daily question at 8:01AM 
+  cronJob = cron.schedule('01 8 * * *', () => {
```

Comment:
add a command to check the status of `cronJob`, whether it is already scheduled or stopped or does not exist yet.

Reply:
Created new variable to hold the status, cronStatus